src/v/storage/tests/storage_e2e_test.cc

```diff
@@ -1062,6 +1062,6 @@ FIXTURE_TEST(check_segment_size_jitter, storage_test_fixture) {
     std::sort(sizes.begin(), sizes.end());
     auto end = std::unique(sizes.begin(), sizes.end());
     auto unique_cnt = std::distance(sizes.begin(), end);
-    // check if all logs have unique segment size
-    BOOST_REQUIRE_EQUAL(unique_cnt, sizes.size());
+    // check if segment sizes are unique
+    BOOST_REQUIRE_GT(unique_cnt, 1);
```

Comment:
would it be sufficient to assert that not all of the sizes are the same?

```
BOOST_ASSERT_FALSE(std::all_of(sizes, [size = sizes[0]](auto other) { return size == other; }));
```

Reply:
right i like this too. 

i assume that we should add a base_size too, so it should be 

`size == other && other >= base_size`

Reply:
jitter can also make segment size smaller so condition `other >= base_size` isn't valid

Reply:
i take it back it is +-5%